src/main/java/controllers/LoginController.java

```diff
@@ -16,8 +17,8 @@
             + "1 Login";
     private InOut inOut;
 
-    public LoginController(InOut inOut, LoginManager loginManager) {
-        this.loginManager = loginManager;
+    public LoginController(InOut inOut, DatabaseManager dbManager) {
+        this.loginManager = new LoginManager(dbManager);
```

Comment:
I'm a bit confused because aren't we supposed to only have one instance of LoginManager in the program? are you making a new one in every controller?